fury/shaders/orient_to_rgb.glsl

```diff
@@ -1,8 +1,3 @@
-in vec3 center;
-in vec3 diff;
-
-out vec3 centerVertexMCVSOutput;
-
 vec3 orient2rgb(vec3 v)
```

Comment:
This file should be in a folder determined by its function. Perhaps, `fury/shaders/utils`.